geonetwork/variant-postgres.md

```diff
@@ -8,28 +8,30 @@ In order to setup the connection from geonetwork, you **must** inject the follow
 
 If your postgres instance is listening on a non-standard port, you must also set that variable: - `POSTGRES_DB_PORT`: postgres port on your database server (defaults to `5432`)
 
-### Linking to a postgres container
+### Connecting to a postgres database
 
-Linking to a postgres container, is pretty straightforward: - `--link <some-postgres-container>:postgres`
+If you want to connect to a postgres server, you need to pass an extra environment variable, containing the IP address for this server (which could be localhost, if you are running it locally). - `POSTGRES_DB_HOST`: IP address of your database server
```

Comment:
I believe `localhost` would be a bad option to use since that would mean `localhost` relative to the geonetwork container and thus not connect to any postgres instance.

Reply:
Perhaps you meant the something running directly on the docker host? That would need to be explicitly pointed out to use the IP address of the docker host (or maybe the OSX or Windows host IP address outside the VM).  It gets complicated fairly quickly and is easier to focus on other containers rather than getting into the networking of everything outside the containers.

Reply:
@yosifkit I agree with you that using "localhost" for networking is "asking for trouble"... my idea was only to tell users that they can connect to an existing database server somewhere (which does not necessarily needs to run on docker), not to explain them how to run a postgres instance locally and then connect to it using its ip address. I [updated the documentation](https://github.com/GeoCat/docs/commit/8e9fed3c072b58a580c8a070fe975203e44f3a93) to reflect this. Now I present only two simple use cases: 1) connect to an external db using DNS 2) connect to a container running on the same docker host, using the container name.